docs/docs/articles/github-actions.md

```diff
@@ -1,16 +1,16 @@
 # Testkube GitHub Action
 
 The Testkube GitHub Action installs Testkube and enables running any [Testkube CLI](https://docs.testkube.io/cli/testkube) command in a GitHub workflow. It is available on Github Marketplace <https://github.com/marketplace/actions/testkube-action>.
-The action provides a flexible way to work with your pipeline and can be used with Testkube Cloud, Testkube Enterprise, and an open source Testkube platform.
+The action provides a flexible way to work with your pipeline and can be used with Testkube Pro, Testkube Enterprise, and an open source Testkube platform.
 
-## Testkube Cloud
+## Testkube Pro
 
-### How to configure Testkube CLI action for TK Cloud and run a test
+### How to configure Testkube CLI action for Testkube Pro and Run a Test
 
-To use this GitHub Action for the [Testkube Cloud](https://cloud.testkube.io/), you need to create [API token](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
+To use this GitHub Action for the [Testkube Pro](https://pro.testkube.io/), you need to create an [API token](https://docs.testkube.io/testkube-cloud/articles/organization-management/#api-tokens).
```

Comment:
shouldn't this be https://app.testkube.io?

Reply:
good catch, will update these